creator-node/scripts/run-tests.sh

```diff
@@ -26,9 +26,6 @@ export logLevel='info'
 # So, if tests are run locally, run docker exec command. Else, run the psql command in the job.
 if [ -z "${isCIBuild}" ]; then
   docker exec -i cn1_creator-node-db_1 /bin/sh -c "psql -U postgres -tc \"SELECT 1 FROM pg_database WHERE datname = 'audius_creator_node_test'\" | grep -q 1 || psql -U postgres -c \"CREATE DATABASE audius_creator_node_test\""
-elif [ -x "$(command -v psql)" ]; then
```

Comment:
what's reason for removing this?

Reply:
this command actually doesn't get executed in the circle container because psql doesn't exist (checked the circle logs). I believe a separate psql container gets spun up